autospec/avxjudge.py

```diff
@@ -0,0 +1,512 @@
+#!/usr/bin/python3
+import subprocess
+import sys
+import re
+import argparse
+import os
+
+# MMX and SSE2 instructions
+sse_instructions_xmm = list()
+
+# 0.1 value instructions
+avx2_instructions_lv = list()
+avx2_instructions_ymm = list()
+avx512_instructions_lv = list()
+
+
+# 1.0 value instructions
+avx2_instructions = list()
+avx512_instructions = list()
+
+# 2.0 value instructions
+avx2_instructions_hv = list()
+avx512_instructions_hv = list()
+
+
+
+
+sse_functions = dict()
+avx2_functions = dict()
+avx512_functions = dict()
+sse_functions_ratio = dict()
+avx2_functions_ratio = dict()
+avx512_functions_ratio = dict()
+
+
+verbose: int = 0
+quiet: int = 0
+
+
+
+
+def init_ins() -> None:
+
+
+    sse_instructions_xmm.append("paddb")
```

Comment:
Do not append every single value... use `extend` instead.

Better than that, just set the `sse_instructions_xmm` list to a literal list like
```python
sse_instructions_xmm = [
    "one",
    "two",
    ...
]
```